packages/zwave-js/src/lib/commandclass/UserCodeCC.ts

```diff
@@ -262,6 +261,38 @@ function persistUserCode(
 					UserIDStatus.Messaging,
 					UserIDStatus.PassageMode,
 			  ]);
+	if (!valueDB.hasMetadata(statusValueId)) {
+		valueDB.setMetadata(statusValueId, {
+			...ValueMetadata.Number,
+			label: `User ID status (${userId})`,
+			states: enumValuesToMetadataStates(
+				UserIDStatus,
+				supportedUserIDStatuses,
+			),
+		});
+	}
+	const codeMetadata: ValueMetadata = {
+		...(typeof userCode === "object"
+			? ValueMetadata.Buffer
+			: ValueMetadata.String),
```

Comment:
```suggestion
		...(Buffer.isBuffer(userCode)
			? ValueMetadata.Buffer
			: ValueMetadata.String),
```
since I tend to stay away from `typeof userCode === "object"`

Reply:
i like this. 👍